app/components/dashboard/about/QuotesExhibition.js

```diff
@@ -24,24 +23,6 @@ export default function QuotesExhibition() {
               unoptimized
             />
           </div>
-
-          {QUOTES.map((quote) => {
```

Comment:
where are you getting the quotes then?

Reply:
I don't need to. I will not include quotes in this component (for now)